NDB.Covid19/NDB.Covid19.iOS/Info.plist

```diff
@@ -104,6 +104,7 @@
 		<string>nn</string>
 		<string>pl</string>
 		<string>so</string>
+		<string>lt</string>
```

Comment:
Rename to "Lit". E.g. if they want "Letvia" (lt / let) later on

Reply:
Don’t really get it. We’ve used two letter ISO codes for every other language, and Latvian is lv. I also think using something else will cause issues with listing supported languages in app stores.

Reply:
Fair point @jonasofrich 
Don't implement my changes then.